doc/source/io.rst

```diff
@@ -4515,6 +4517,69 @@ Read from a feather file.
    import os
    os.remove('example.feather')
 
+
+.. _io.parquet:
+
+Parquet
+-------
+
+.. versionadded:: 0.20.0
+
+Parquet provides a sharded binary columnar serialization for data frames. It is designed to make reading and writing data
```

Comment:
The main difference between Feather and Parquet is that Feather stores the data as-it's-in-memory whereas Parquet uses a variety of encoding and compression techniques to shrink the file size as much as possible while still maintaining good read performance.

Furthermore, Parquet adds metadata so that you can query the files efficiently ("predicate pushdown").